c7n/cli.py

```diff
@@ -73,9 +73,13 @@ def _default_options(p, blacklist=""):
     config.add_argument("-t", "--resource", default=None, dest='resource_type',
                         help="Only use policies with the given resource type")
 
-    p.add_argument("-v", "--verbose", action="store_true",
-                   help="Verbose logging")
-    p.add_argument("--debug", default=False, help=argparse.SUPPRESS,
+    config.add_argument("-v", "--verbose", action="count", help="Verbose logging")
+    if 'quiet' not in blacklist:
```

Comment:
rather than blacklist across a majority of cmds in generic options, just add it to the specific options it applies to explicitly,